test-requirements.txt

```diff
@@ -13,3 +13,5 @@ dnspython<2.0
 newrelic
 executing
 asttokens
+chalice
```

Comment:
Please put this stuff into tox.ini, i.e. create a new job for your integration (+ testing different versions you want to support)

Reply:
done!